### PR TITLE
mfoc: use github url, add head, fix install

### PR DESCRIPTION
**Pull request by niki-timofe**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by niki-timofe** [comment]
Review please, someone

File: Formula/mfoc.rb [hunk 1]
```diff
@@ -19,11 +20,14 @@ class Mfoc < Formula
...
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "pkg-config" => :build
   depends_on "libnfc"
   depends_on "libusb"
 
   def install
+    system "autoreconf", "-is"
```
Comment (SMillerDev) [inline Formula/mfoc.rb:30]: Why?

File: Formula/mfoc.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
 class Mfoc < Formula
   desc "Implementation of 'offline nested' attack by Nethemba"
   homepage "https://github.com/nfc-tools/mfoc"
-  url "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/mfoc/mfoc-0.10.7.tar.bz2"
-  sha256 "93d8ac4cb0aa6ed94855ca9732a2ffd898a9095c087f12f9402205443c2eb98c"
-  license "GPL-2.0"
-  revision 1
+  url "https://github.com/nfc-tools/mfoc/archive/refs/tags/mfoc-0.10.7.tar.gz"
+  sha256 "2dfd8ffa4a8b357807680d190a91c8cf3db54b4211a781edc1108af401dbaad7"
```
Comment (SMillerDev) [inline Formula/mfoc.rb:5]: Is there any official confirmation of this?

Comment (niki-timofe) [reply Formula/mfoc.rb:5]: https://code.google.com/archive/p/mfoc/

File: Formula/mfoc.rb (hunk 1 above)
Comment (niki-timofe) [reply Formula/mfoc.rb:30]: As described in readme https://github.com/nfc-tools/mfoc#build-from-source

**Comment by SMillerDev** [comment]
Could you suggest a command we can run as a test that is a little more indicative of failures than `--help`?

**Comment by niki-timofe** [comment]
@SMillerDev, it won't actually work without supported NFC reader, but at least, you can see a message, indicating that reader was not found:
```
mfoc -O /dev/null
```

**Review by SMillerDev — APPROVED** [review]
Thanks @niki-timofe   ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**